lib/decode.js

```diff
@@ -55,12 +61,12 @@ function decode (data) {
         typetags = typetags.value;
         // so we start building our message list
         if (typetags[0] !== ',') {
-            throw 'invalid type tag in incoming OSC message, must start with comma';
+            throw new Error('invalid type tag in incoming OSC message, must start with comma');
```

Comment:
modifying errors is technically a semver major change while the other changes are all semver minor. I think this should likely be moved to it's own PR to be discussed on it's own merit